bench/irmin-pack/tree.ml

```diff
@@ -16,10 +16,39 @@ type op =
   | Copy of key * key
 [@@deriving yojson]
 
+type config = {
+  ncommits : int;
+  ncommits_trace : int;
+  depth : int;
+  nchain_trees : int;
+  width : int;
+  nlarge_trees : int;
+  root : string;
+  flatten : bool;
+  inode_config : [ `Entries_32 | `Entries_2 ];
+  store_type : [ `Pack | `Pack_layered ];
+  freeze_commit : int;
```

Comment:
maybe you can combine the two: it does not make sense to have a `freeze_commit` if the store type is not layered.

Reply:
This data-structure already mixes things (e.g. `flatten` and `ncommits_trace` only apply to trace mode). Adding `freeze_commit` here is consistent with what we have been doing so far in this file.

We could clean this in a latter patch